### PR TITLE
Update example ConsoleTemplate metadata

### PR DESCRIPTION
**Pull request by craigfurman**
~~Add a namespace~~, and change labels to repo+env as currently used by the
only consumer to search for ConsoleTemplates.

### CODE REVIEW

**Comment by craigfurman** [comment]
I suspect the CircleCI acceptance test step is broken, as this works for me* in a clean KinD cluster.

\*: 2019 equivalent of "it works on my machine"

File: config/samples/workloads_v1alpha1_consoletemplate.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ spec:
       restartPolicy: Never
 metadata:
   name: console-template-0
+  namespace: default
   labels:
-    app: myapp
-    release: myapp-sandbox
+    repo: myapp-owner-myapp-repo
+    env: myapp-env
```
Comment (lawrencejones) [inline config/samples/workloads_v1alpha1_consoletemplate.yaml]: We've recently changed our label taxonomy to use `environment` everywhere. What are your thoughts on env vs environment here?

Comment (craigfurman) [reply config/samples/workloads_v1alpha1_consoletemplate.yaml]: Not particularly fussed, will change

File: config/samples/workloads_v1alpha1_consoletemplate.yaml [hunk 2]
```diff
@@ -22,6 +22,7 @@ spec:
       restartPolicy: Never
 metadata:
   name: console-template-0
+  namespace: default
```
Comment (benwh) [inline config/samples/workloads_v1alpha1_consoletemplate.yaml]: Why add namespace here?

Comment (craigfurman) [reply config/samples/workloads_v1alpha1_consoletemplate.yaml]: `namespace: default` is redundant, but in practice we will add namespaces to ConsoleTemplates, which the Console will inherit.

Comment (benwh) [reply config/samples/workloads_v1alpha1_consoletemplate.yaml]: I still don't see that it adds any value in this example - this is obviously a namespaced resource so the object will have to go in *some* namespace. Adding this here just introduces the downside that if you were to pipe this through `kubectl apply` while targeting a different namespace in your current config, then the resource isn't going to go where you'd expect.

Comment (craigfurman) [reply config/samples/workloads_v1alpha1_consoletemplate.yaml]: > if you were to pipe this through `kubectl apply` while targeting a different namespace in your current config, then the resource isn't going to go where you'd expect.

I was about to reply with "That isn't true" but it depends on what you mean by "expect"? I don't expect kubectl apply to only modify resources in my namespace, it depends on what's in the resource definitions.

In any case I'm not too fussed, I'll remove the namespace.

**Pull request merged**